authors/urls.py

```diff
@@ -25,6 +25,6 @@
 
     path('api/', include('authors.apps.authentication.urls', )),
     path('swagger/', schema_view),
-    path('docs/', coreapi_docs),
+    path('', coreapi_docs),
```

Comment:
@walimike Please make this url just to redirect to the coreapi docs, so you should maintain both the root url and docs url.
See the rest framework documentation on how to do redirects.